xml/System.Net.Security/SslClientAuthenticationOptions.xml

```diff
@@ -206,8 +231,8 @@
         <ReturnType>System.Security.Authentication.SslProtocols</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets the <see cref="T:System.Security.Authentication.SslProtocols" /> value that represents protocols used for authentication.</summary>
```

Comment:
We should make sure IMHO that this is set of protocols client wants to use. One of them may be used or negotiation may fail.

Reply:
This is same for SslServerAuthemticationOptions hence I put it there: https://github.com/aik-jahoda/dotnet-api-docs/blob/27ea1dbfb6dd106749d7d4c9fda77ebf0ed4429b/xml/System.Security.Authentication/SslProtocols.xml#L45

Reply:
You will get that only if you follow the link but not oof you joust read the property decryption, right? 
how about
> Gets or sets the <see cref="T:System.Security.Authentication.SslProtocols" /> value that represents protocol  versions offered by client to the server during authentication.

Reply:
I updated this summary also in SslServerAuthenticationOptions.xml